xml/System.Net/WebClient.xml

```diff
@@ -411,7 +409,10 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- If an operation is pending, this method calls <xref:System.Net.WebRequest.Abort%2A> on the underlying <xref:System.Net.WebRequest>.  
+ If an operation is pending, this method calls <xref:System.Net.WebRequest.Abort%2A> on the underlying <xref:System.Net.WebRequest>.
+ 
+ > [!NOTE]
+ > Starting in .NET Core 2.0, <xref:System.Net.WebClient.CancelAsync%2A> doesn't cancel the request immediately if the response has started to fetch. If the response has started to fetch, the request is cancelled only after a complete response is read.
```

Comment:
I would motivate people for the transition to HttpClient here and be more explicit as the current behavior looks buggy.
Something like:

`For proper cancelation behavior use HttpClient instead of WebClient`

@gewarren, @karelz what do you think?

Reply:
Sounds good to me. Thanks!

Reply:
Thanks!